src-js/components/TicksPage.jsx

```diff
@@ -39,10 +41,10 @@ export default class TicksPage extends React.Component {
         let relevantTicks;
         if (routeType === 'Boulder') {
             relevantTicks = ticks.filter((tick) =>
-                ((tick['route-type'] === routeType) && (tick['style'] !== "Attempt")));
+                ((tick['route-type'] === routeType) && (tick['climb-style'] !== "Attempt")));
         } else if (routeType === 'Lead') {
             relevantTicks = ticks.filter((tick) =>
-                ((tick['style'] === "Lead") && (tick['lead-style'] !== "Fell/Hung")));
+                ((tick['climb-style'] === "Lead") && (tick['lead-style'] !== "Fell/Hung")));
```

Comment:
We could consider normalizing this data on the backend so that all ticks have a somewhat consistent naming scheme. out of scope for this CR.